<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Filters...</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QCheckBox" name="cbDescription">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="cbLogbook">
       <property name="text">
        <string>Logbook</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="cbTag">
       <property name="text">
        <string>Tag</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="cbID">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="leID">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDescription">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLogbook">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTag">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbID</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDescription</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLogbook</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTag</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leID</sender>
   <signal>returnPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLogbook</sender>
   <signal>returnPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDescription</sender>
   <signal>returnPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTag</sender>
   <signal>returnPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
